src/main.cpp

```diff
@@ -1996,11 +1994,12 @@ bool CheckInputs(const CTransaction& tx, CValidationState &state, const CCoinsVi
                 assert(coins);
 
                 // Verify signature
-                CScriptCheck check(*coins, tx, i, flags, cacheStore);
-                if (pvChecks) {
-                    pvChecks->push_back(CScriptCheck());
-                    check.swap(pvChecks->back());
-                } else if (!check()) {
+                if (emplacer) {
+                    emplacer(CScriptCheck(*coins, tx, i, flags, cacheStore));
+                    continue;
```

Comment:
I tend to prefer else over continue, but it is understandable is just to avoid the re-indent and minimize disruption.
I would the continue be gone "for free" when/if  CheckInputs needs a reindent (btw, unrelated, we've been owing one to Consensus::CheckTxInputs for a while...)